pkgs/tools/networking/network-manager/default.nix

```diff
@@ -11,11 +11,11 @@ let
   pythonForDocs = python3.withPackages (pkgs: with pkgs; [ pygobject3 ]);
 in stdenv.mkDerivation rec {
   name = "network-manager-${version}";
```

Comment:
pname <3 !

Reply:
ha pname and name differ. Shouldn't the name be `NetworkManager` instead ?

Reply:
>  Shouldn't the name be NetworkManager instead ?

Nope, nixpkgs manual says derivation names shouldn't be formatted that way.